roboteam_ai/src/skills/Pass.cpp

```diff
@@ -23,14 +23,20 @@ Pass::Status Pass::onUpdate() {
 
     switch(currentProgress) {
         case Progression::POSITIONING: {
-            if (!coach::Coach::isRobotBehindBallToPosition(0.30, robotToPassTo->pos, robot->pos)) {
-                goToType = GoToType::NUMERIC_TREES;
-                targetPos = Coach::getPositionBehindBallToPosition(0.30, robotToPassTo->pos);
-            } else if (!World::ourBotHasBall(robot->id)) {
+            if (!coach::Coach::isRobotBehindBallToPosition(0.40, robotToPassTo->pos, robot->pos)) {
+                std::cout << "Going behind ball" << std::endl;
```

Comment:
There are print statements which should be removed. Also... why did you change passing in this PR?

Reply:
Done. I wanted to test the positioning with passing, but it wasn't working properly so I had to change some things

Reply:
Done. I wanted to test the positioning using passing, which wasn't working properly then, so I changed it